website/docs/static.md

```diff
@@ -47,7 +47,7 @@ Each strictness level reports all errors at lower levels, plus new errors:
   all are reported in that file. **Note**: ignoring a file can cause errors to
   appear in _other_ files, because that other file references something defined
   in an ignored file. We recommend pushing the entire project to out of `ignore`
-  (at Stripe, 100% of non-test files are not ignored.)
+  (at Stripe, no files are ignored.)
```

Comment:
I am...not sure I believe this (e.g. it looks like some of our generated proto code is `typed: ignore`.

Reply:
yeah i was about to reword that, then forgot. those files are autogenerated, and we have generated RBI files from protoc-gen-rbi that strictly type all the definitions that would have been generated. I can just put it back to non-test which is technically correct. maybe i'll find a way to rewrite this whole passage.